src/v2/shared/routes.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+import UriTemplate from 'uri-templates';
+
+const apiKey = '{?apiKey}';
+
+const template = (path) => UriTemplate(path + apiKey);
+
+export const Kind = {
+  COLLECTION: 'COLLECTION',
+  ID: 'ID',
+  APPLIES_TO: 'APPLIES_TO',
+  WORKFLOW: 'WORKFLOW',
+  DATE: 'date',
+  CHECK: 'check'
+};
+
+const Routes = {
+  fees: {
+    collection: template('/hub/v2/pmcs/{pmcId}/fees'),
+    id: template('/hub/v2/pmcs/{pmcId}/fees/{feeId}'),
+    appliesTo: template('/hub/v2/pmcs/{pmcId}/fees/{feeId}/applies-to'),
+    workflow: template('/hub/v2/pmcs/{pmcId}/fees/workflows/{workflowId}')
+  },
+
+  media: {
+    collection: template('/hub/v2/pmcs/{pmcId}/units/{unitId}/media'),
+    id: template('/hub/v2/pmcs/{pmcId}/units/{unitId}/media/{mediaId}'),
+    workflow: template('/hub/v2/pmcs/{pmcId}/units/{unitId}/media/workflows/{workflowId}')
+  },
+
+  pmcs: {
+    collection: template('/hub/v2/pmcs'),
+    id: template('/hub/v2/pmcs/{pmcId}'),
+    workflow: template('/hub/v2/pmcs/workflows/{workflowId}')
+  },
+
+  policies: {
+    collection: template('/hub/v2/pmcs/{pmcId}/policies'),
+    id: template('/hub/v2/pmcs/{pmcId}/policies/{policyId}'),
+    appliesTo: template('/hub/v2/pmcs/{pmcId}/policies/{policyId}/applies-to'),
```

Comment:
Most of the routes work I got from here: https://public-api.apis.leisurelink.com/docs/hub#/